Before: aa5716e985baeb09120aba9371889e32e26d261c1d28850ce5963f7f98149be4
After: f3838f4e7116ec64801ae19321103deb8f58b63008869fdfb744deea4f82503c
Patch: 45177e47390f5d2a751c9b2739e466cb1698a777f4c0615192842ffab850dae2
--- Guardfile	2012-09-16 16:33:53.000000000 +1000
+++ Guardfile	2012-09-12 12:04:30.000000000 +1000
@@ -1,7 +1,5 @@
 require 'corvid/builtin/guard'
-
-project_name  = Regexp.quote(determine_project_name)
-rspec_options = read_rspec_options(File.dirname __FILE__)
+rspec_options= read_rspec_options(File.dirname __FILE__)
 
 ignore VIM_SWAP_FILES
 
@@ -12,8 +10,7 @@
   guard 'minitest', test_folders: 'test/unit', test_file_patterns: '*_test.rb' do
 
     # Src files
-    watch(%r'^lib/(.+)\.rb$')                 {|m| "test/unit/#{m[1]}_test.rb"}
-    watch(%r'^lib/#{project_name}/(.+)\.rb$') {|m| "test/unit/#{m[1]}_test.rb"}
+    watch(%r'^lib/(.+)\.rb$') {|m| "test/unit/#{m[1]}_test.rb"}
 
     # Each test
     watch(%r'^test/unit/.+_test\.rb$')
@@ -34,8 +31,7 @@
   guard 'rspec', binstubs: true, spec_paths: ['test/spec'], cli: rspec_options, all_on_start: false, all_after_pass: false do
 
     # Src files
-    watch(%r'^lib/(.+)\.rb$')                 {|m| "test/spec/#{m[1]}_spec.rb"}
-    watch(%r'^lib/#{project_name}/(.+)\.rb$') {|m| "test/spec/#{m[1]}_spec.rb"}
+    watch(%r'^lib/(.+)\.rb$') {|m| "test/spec/#{m[1]}_spec.rb"}
 
     # Each spec
     watch(%r'^test/spec/.+_spec\.rb$')
--- test/spec/%plugin_name%_plugin_spec.rb.tt	2012-09-20 11:18:09.000000000 +1000
+++ test/spec/%plugin_name%_plugin_spec.rb.tt	2012-09-11 16:52:34.000000000 +1000
@@ -4,9 +4,9 @@
 require '<%= require_path %>'
 
 describe <%= full_class_name %> do
-  include Corvid::Builtin::ResourcePatchTests
+  include Corvid::ResourcePatchTests
 
   include_resource_patch_tests
 
-  include_feature_update_tests
+  include_feature_update_install_tests
 end
